### PR TITLE
Add wheels to FPF PyPi for latest securedrop-sdk

### PR DESCRIPTION
**Pull request by creviera**
# Description

The new package and sha256sum.txt file need a signature and https://github.com/freedomofpress/securedrop-debian-packaging/issues/203 is something that came up during build.

Other than that, this PR includes the output of the build-wheels script for the latest sdk 0.2.0.

### CODE REVIEW

**Comment by creviera** [comment]
See build logs: https://github.com/freedomofpress/build-logs/blob/main/workstation/securedrop-sdk-0.2.0-20201026

**Comment by creviera** [comment]
https://github.com/freedomofpress/securedrop-debian-packaging/issues/203 is no longer blocking on this and can be marked as resolved once we decide on how to move forward for future releases, see https://github.com/freedomofpress/securedrop-debian-packaging/issues/203#issuecomment-717413074

@emkll this should be ready to be signed

**Comment by eloquence** [comment]
(Needs rebase)

**Review by emkll — APPROVED** [review]
Thanks @creviera changes look good to me. I've rebased on latest main due to merge of #198 and:
- appended a detached signature for the changes you provided
- updated index.html (this is still a manual step)

File: sha256sums.txt [hunk 1]
```diff
@@ -42,6 +42,8 @@ c8d98208fb2074336c06be3fef0994a8a57fde7a765cead12bc36e9128d319e2  securedrop_sdk
 970fde25e6238e1808ac120951ee972549f4cd7952966dfe29f731bb308cc0d8  securedrop-sdk-0.1.0.tar.gz
 a631495acd741ab568410287879c5a3af3ccd38e00a2f3a127cc6b27cba99392  securedrop_sdk-0.1.1-py3-none-any.whl
 138ce7a717db519c3c8d19b9475d7660fb7095d7608e8802723e682a7415e677  securedrop-sdk-0.1.1.tar.gz
+dd0231e1b873197373a7e84e3610370634eead601223551153eca7cdd166e7d5  securedrop_sdk-0.2.0-py3-none-any.whl
+c4a343077e8c0a38914e17f6369b830f1e361f9d66699b20803c07b39472357f  securedrop-sdk-0.2.0.tar.gz
```
Comment (emkll) [inline sha256sums.txt:46]: :heavy_check_mark:

**Pull request merged**